doc/languages-frameworks/index.md

```diff
@@ -0,0 +1,31 @@
+# Languages and frameworks {#chap-language-support}
+
+The standard build environment makes it easy to build typical Autotools-based packages with very little code. Any other kind of package can be accomodated by overriding the appropriate phases of `stdenv`. However, there are specialised functions in Nixpkgs to easily build packages for other programming languages, such as Perl or Haskell. These are described in this chapter.
+
+* [Agda](agda.html)
```

Comment:
Hardest part are the anchors. You will want something independent from output format and decoupled from file structure. That way you can render it to anything, including a single-page HTML. This is where document-oriented tools like Docbook with document-wide identifiers shine and tools that are file-based have to resort to ugly hacks.

Reply:
Okay, I'll tackle this issue next.

Reply:
I have a new anchor system now. There is a anchors.nix where you associate an anchor with a file. Right now it only supports HTML but I'll update that to be more generic.